### PR TITLE
New runtime

### PR DESCRIPTION
**Pull request by bztd**

### CODE REVIEW

**Comment by bztd** [comment]
@bbhtt I have tried to compile version 6.0 with the same manifest but it failed, identically to version 5.10.1. I have tried running some example scripts and they work fine. I think this version should be merged while someone tries to get 6.0 working.

File: flatpak_options.py.patch [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ 
+ cxx_extra='-w -O3'
+ 
+-pythoncmd='/app/bin/python3'
+-pythonlibpath = '/app/lib'
+-pythonincpath = '/app/include/python3.7m'
++pythoncmd='python3'
++pythonlibpath = '/app/lib/python3.10'
++pythonincpath = '/app/include/python3.10'
++pythonlibname='python3.10'
```
Comment (bbhtt) [inline flatpak_options.py.patch:13]: 23.08 has python 3.11

File: au.edu.uq.esys.escript.json [hunk 2]
```diff
@@ -194,53 +146,39 @@
...
-                "mkdir -p /app/share/icons/hicolor/128x128/apps/",
-                "mkdir -p /app/share/icons/hicolor/256x256/apps/",
-                "mkdir -p /app/share/applications/",
-                "mkdir -p /app/share/appdata",
-                "cp ./tools/flatpak/64x64_au.edu.uq.esys.escript.png /app/share/icons/hicolor/64x64/apps/au.edu.uq.esys.escript.png",
-                "cp ./tools/flatpak/128x128_au.edu.uq.esys.escript.png /app/share/icons/hicolor/128x128/apps/au.edu.uq.esys.escript.png",
-                "cp ./tools/flatpak/256x256_au.edu.uq.esys.escript.png /app/share/icons/hicolor/256x256/apps/au.edu.uq.esys.escript.png",
-                "cp ./tools/flatpak/au.edu.uq.esys.escript.desktop /app/share/applications/au.edu.uq.esys.escript.desktop",
-                "cp ./tools/flatpak/au.edu.uq.esys.escript.appdata.xml /app/share/appdata/au.edu.uq.esys.escript.appdata.xml",
+                "ln -s /app/lib/python3.10/site-packages/numpy/core/include/numpy /app/include/",
```
Comment (bbhtt) [inline au.edu.uq.esys.escript.json:158]: Same here python 3.11

File: flatpak_options.py.patch (hunk 1 above)
Comment (bztd) [reply flatpak_options.py.patch:13]: Using this version breaks some dependencies. I can try to compile version 6.0 with python 3.11, but it's hard to know if I'll be successful. I think this version should be merged anyway.

**Comment by bbhtt** [comment]
Please write more descriptive commit messages and put related changes in the same commit in the future. Otherwise I will have to squash everything into one commit. I don't always have the time to rewrite commits.

**Pull request merged**